eultheme/managers.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from django.db import models
+from django.db.models import Q
+from django.conf import settings
+from django.utils.timezone import utc
+
+class BannerQuerySet(models.query.QuerySet):
+
+    def active(self):
```

Comment:
A brief comment/description here would be helpful.  (i.e., why is active more than just not disabled?)